spec/Formatter/IntlMoneyFormatterSpec.php

```diff
@@ -24,13 +25,13 @@ function it_is_a_money_formatter()
         $this->shouldImplement(MoneyFormatter::class);
     }
 
-    function it_formats_money(\NumberFormatter $numberFormatter)
+    function it_formats_money(\NumberFormatter $numberFormatter, CurrenciesWithSubunit $currenciesWithSubunit)
     {
-        $numberFormatter->getAttribute(\NumberFormatter::FRACTION_DIGITS)->willReturn(2);
-        $numberFormatter->formatCurrency('0.01', 'EUR')->willReturn('€1.00');
-
         $money = new Money(1, new Currency('EUR'));
 
+        $numberFormatter->formatCurrency('0.01', 'EUR')->willReturn('€1.00');
+        $currenciesWithSubunit->getSubunitsFor($money->getCurrency())->willReturn(2);
```

Comment:
Hm, I am not sure what is gramatically correct: subunit or subunitS. The first seems to be more logical to me. Can you check which is correct?